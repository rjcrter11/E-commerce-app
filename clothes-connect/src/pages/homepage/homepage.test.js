import React from 'react';
import { create } from 'react-test-renderer';
import { shallow } from 'enzyme';
import HomePage from './homepage.component';

it('should render HomePage coponent', () => {
    const wrapper = shallow(<HomePage />)
    expect(wrapper).toMatchSnapshot();
})
