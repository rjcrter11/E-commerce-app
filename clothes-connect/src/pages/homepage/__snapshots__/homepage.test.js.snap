// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage coponent 1`] = `ShallowWrapper {}`;
